pynetgear/__init__.py

```diff
@@ -110,14 +113,22 @@ def login(self):
         """
         # cookie is also used to track if at least
         # one login attempt has been made for v1
+        if self._logging_in:
```

Comment:
can you move that bloc up before the `cookie` comment so the comment stays nearby the commented var ?

Reply:
done